src/components/EventForm/Form.js

```diff
@@ -171,7 +171,8 @@ export default function Form(props) {
                     radioGroupTitle={"Faculty"}
                     {...props}/>  
                     <RadioGroup onChange={(e) => {toggleOtherState(e)}}>
-                        {createButtons(["ANDYSTEST","Commerce","Science","Forestry", "Other:"])}
+                        {createButtons(["ANDYSTEST","Commerce","Science","Forestry"])}
```

Comment:
Instead of a function can you make this a new component? with props `buttonOptions` and `otherOptionEnabled`

example `<ButtonGroup buttonOptions={["ANDYSTEST","Commerce","Science","Forestry"]} otherOptionEnabled={true} />`

Reply:
Formik props need to be passed in as well, and I'm not sure if formik will behave properly if this is the case. Will attempt this and get back with results (if it's too complex, I'll just leave it for now).

Reply:
So things work but not well. The main issue is the formik validation for the 'other' option requires different id's for every validation group. Wanted to consult you on how my stuff is desigined before i go too far in a potentially non-ideal direction.

Reply:
What's your question?